debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -0,0 +1,94 @@
+package io.debezium.connector.sqlserver;
+
+import java.sql.SQLException;
+import java.util.Objects;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.IoUtil;
+
+/**
+ * {@link JdbcConnection} extension to be used with Microsoft SQL Server
+ * 
+ * @author Horia Chiorean (hchiorea@redhat.com)
+ */
+public class SqlServerConnection extends JdbcConnection {
+    
+    private static final String URL_PATTERN = "jdbc:sqlserver://${" + JdbcConfiguration.HOSTNAME + "}:${"
+                                              + JdbcConfiguration.PORT + "};databaseName=${" + JdbcConfiguration.DATABASE + "}";
+    protected static ConnectionFactory FACTORY = JdbcConnection.patternBasedFactory(URL_PATTERN,
+                                                                                    com.microsoft.sqlserver.jdbc.SQLServerDriver.class.getName(),
+                                                                                    SqlServerConnection.class.getClassLoader());
+    private static Logger LOGGER = LoggerFactory.getLogger(SqlServerConnection.class);
+    
+    private static final String STATEMENTS_PLACEHOLDER = "#";
+    private static final String ENABLE_DB_CDC;
+    private static final String ENABLE_TABLE_CDC;
+    private static final String CDC_WRAPPERS_DML;
+
+    static {
+        try {
+            Properties statements = new Properties();
+            ClassLoader classLoader = SqlServerConnection.class.getClassLoader();
+            statements.load(classLoader.getResourceAsStream("statements.properties"));
+            ENABLE_DB_CDC = statements.getProperty("enable_cdc_for_db");
+            ENABLE_TABLE_CDC = statements.getProperty("enable_cdc_for_table");
+            CDC_WRAPPERS_DML = IoUtil.read(classLoader.getResourceAsStream("generate_cdc_wrappers.sql"));
+        } catch (Exception e) {
+            throw new RuntimeException("Cannot load SQL Server statements", e);
+        }
+    }
+    
+    /**
+     * Creates a new connection using the supplied configuration.
+     *
+     * @param config {@link Configuration} instance, may not be null.
+     */
+    public SqlServerConnection(Configuration config) {
+        super(config, FACTORY);
+    }
+    
+    /**
+     * Returns a JDBC connection string for the current configuration.
+     *
+     * @return a {@code String} where the variables in {@code urlPattern} are replaced with values from the configuration
+     */
+    public String connectionString() {
+        return connectionString(URL_PATTERN);
+    }
+    
+    /**
+     * Enables CDC for a given database, if not already enabled.
+     * 
+     * @param name the name of the DB, may not be {@code null}
+     * @throws SQLException if anything unexpected fails
+     */
+    public void enableDBCDC( String name) throws SQLException {
```

Comment:
Minor nit, but I thought I'd bring it up to converge to a consistent style: s/enableDBCDC/enableDbCdc/.

I.e. only the first letter of an abbreviation should be uppercased in identifiers, e.g. see `DdlParser` (instead of `DDLParser`). Would be nice to adhere to this everywhere.